cinema/models.py

```diff
@@ -5,6 +5,32 @@
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    genres = models.ManyToManyField("Genre", blank=True, related_name="genres")
+    actors = models.ManyToManyField("Actor", blank=True, related_name="actors")
```

Comment:
The `related_name` for the `actors` field in the `Movie` model is set to 'actors', which is not unique or descriptive. Consider using a more descriptive name like 'movies_as_actor' to indicate the reverse relationship.